src/components/ListItem.tsx

```diff
@@ -100,31 +112,47 @@ const DisciplineItem = ({ onPress, icon, title, description, badgeLabel }: Disci
     [pressInY, onPress]
   )
 
+  const titleToRender =
+    typeof title === 'string' ? (
+      <Title pressed={pressed} numberOfLines={3}>
+        {title}
+      </Title>
+    ) : (
+      title
+    )
+
+  const iconToRender = icon ? (
```

Comment:
```suggestion
  const iconToRender = icon ?? (
    <IconContainer>{typeof icon === 'string' ? <Icon source={{ uri: icon }} /> : icon}</IconContainer>
  )
```

Reply:
sorry multiline suggestions are difficult. I think here you could use the same shorthand like line 128